services/shared/serverless.yml

```diff
@@ -0,0 +1,53 @@
+service: shd-monorepo-example # To be replaced
```

Comment:
I'm not sure if this file is necessary. Why don't we just merge this file and the `serverless.common.yml`? I have the same feeling about the `package.json` file in this `/services/shared` folder.